pages/home.md

```diff
@@ -3,5 +3,5 @@ layout: home
 permalink: /
 hero-image: /assets/img/clouds.jpg
 hero-text: "The Government Innovation Platform"
-hero-subtext: "Free your team to focus on creating quality services securely hosted in the cloud"
+hero-subtext: "We built it. We manage it. We keep improving it. Want to use it?"
```

Comment:
It was in James Hupp's document: https://docs.google.com/document/d/1EJEToDGuloXp1GMTqpgdCxLQwF2Z5bopdwGm-UUWMOQ/edit#. It was one of a few options, please pick whichever one you like.